src/tests/test_01.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+#This block of code allows access to the src folder and the files that need to be tested within
+import sys
+sys.path.append(r"C:\Users\jackw\uvic\rocketry\Xenia1-flight-Computer\src")
```

Comment:
write a note in the readme to change this or make a function to auto detect the path (might be hard)